lib/location/location_search_delegate.dart

```diff
@@ -639,18 +639,20 @@ class _SuggestionList extends StatelessWidget {
 
   Future<void> _handleOnYourLocationTapped(BuildContext context) async {
     final localization = TrufiLocalization.of(context);
-    final location = await LocationProviderBloc.of(context).currentLocation;
-    if (location != null) {
-      _handleOnLatLngTapped(
-        description: localization.searchMapMarker,
-        location: location,
-        addToHistory: false,
+    final location =
+        await context.read<LocationProviderCubit>().getCurrentLocation();
+
+    if (location == null) {
+      showDialog(
+        context: context,
+        builder: (context) => buildAlertLocationServicesDenied(context),
       );
```

Comment:
since the location will never be null, I think instead we should handle the throw error.

Reply:
@md-weber  fixed here [PR 434](https://github.com/trufi-association/trufi-core/pull/434)